dotCMS/src/integration-test/java/com/dotmarketing/cms/urlmap/URLMapAPIImplTest.java

```diff
@@ -75,7 +67,9 @@ public void prepareTest() {
     @Test
     public void shouldReturnContentletWhenTheContentExists()
             throws DotDataException, DotSecurityException {
-
+        final String newsPatternPrefix =
+                "/testpattern" + System.currentTimeMillis() + "/";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "/testpattern" appears 4 times in this file; the first occurrence is on line 71](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4543718)